src/partials/advertisement.html

```diff
@@ -1,2 +1,7 @@
-<section class="advertisement">
-  </section>
\ No newline at end of file
+<section class="advertisement ">
+  <div class="container-discount">
+  <h2 class="visually-hidden">Advertisement</h2>
+    <p class="text-discount">Discount up to 40%!<span class="discount-part-text"> Taste the vitamins and keep your health</span></p>
```

Comment:
please change classes name and don't forget to upload classes name in css document